### PR TITLE
[ADD][11.0] - Adding module project_agile_kanban

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello, 

Module ``project_agile_kanban`` which extends module [project_agile](https://github.com/OCA/project-agile/pull/4) and leans to module [project_agile_analytic](https://github.com/OCA/project-agile/pull/6) enables you to manage your projects by using agile kanban methodology.

Kind Regards,
Petar

### CODE REVIEW

File: project_agile_kanban/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_agile_kanban/README.rst]: see https://github.com/OCA/web/pull/881#pullrequestreview-99094143

File: project_agile_kanban/models/project_agile_board_reader.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                    multiples.append(status_name)
+
+                if status_name not in wkf_states:
+                    lost_and_found.add(status_name)
+
+        error_messages = []
+
+        if multiples:
+            error_messages.append(_(
+                "Following states: [%s] are assigned to multiple columns!"
```
Comment (elicoidal) [inline project_agile_kanban/models/project_agile_board_reader.py]: Better not use "!" in error messages

**Review by elicoidal — CHANGES_REQUESTED** [review]
Missing UT + some demo data

File: project_agile_kanban/README.rst (hunk 1 above)
Comment (pnajman-modoolar) [reply project_agile_kanban/README.rst]: Fixed

File: project_agile_kanban/models/project_agile_board_reader.py (hunk 2 above)
Comment (pnajman-modoolar) [reply project_agile_kanban/models/project_agile_board_reader.py]: Fixed

**Pull request closed without merging**